### ISSUE TITLE
view all orders

### ISSUE THREAD

**Issue by edgaleano** [vokomokum/foodsoft-vokomokum#3]
in foodsoft you can see the order of one person but not all orders. this is necessary to support the functionality of:

- sending the order to Odin
- the setup list (seeing which products have and have not arrived)
- accounting

**Comment by wvengen** [comment]
Hi, just looking through your issues! Very nice to see you made a list of things to figure out and improve.

For this issue, I wonder if you've missed parts of Foodsoft, since the things you mention are part of regular foodcoop flows.
Mostly, when look at the order via _Orders_ &gt; _Manage orders_, you can see all products for the order, including all members. There is an option to export it as various kinds of PDFs, and also as CSV.

See e.g. [app.foodcoops.net's demo](https://foodcoops.net/demo/), e.g. [this order](https://app.foodcoops.net/demo/orders/3), you may need to close it. Then you see:

![image](https://github.com/vokomokum/foodsoft-vokomokum/assets/503804/c2b53473-bb99-47d2-8184-87c61ea48853)

"the setup list" seems to suggest the receive functionality in Foodsoft (fresh already uses Foodsoft, so this must be known by someone). Here you can receive an order, and if the amounts received differ, products are redistributed across members.

Accounting is done by _Finances_ &gt; _Account orders_. See also the manual fresh uses, and the [German documentation](https://docs.foodcoops.net/de/home) (sorry, German is most complete here).

**Comment by edgaleano** [comment]
@wvengen yes, we totally missed this functionality :sweat: I think that we didn't test the main version of foodsoft long enough, or that we maybe missed this functionality because we first had to close the order. In either case: I'll close the issue because the functionality is already there!

And thanks for the link to the wiki. it's also really good to know that that resource exists! so far we had only found the wiki link to the github repository
